yt/visualization/volume_rendering/interactive_loop.py

```diff
@@ -84,9 +84,7 @@ def setup_loop(self, scene, camera):
         return callbacks
 
     def start_loop(self, scene, camera):
-        callbacks = self.setup_loop(scene, camera)
```

Comment:
removing dead code here exposes that the method is currently just a direct wrapper for another one

Reply:
cool !